### PR TITLE
Sort student names on grading page

### PR DESCRIPTION
**Pull request by shadaj**
Partial fix towards #1027

### CODE REVIEW

**Comment by bnmnetp** [comment]
This makes the names sorted on the manage sections tab.  Which is a good thing!

The grading page is a bit trickier, I think, but the code for that is in the `pickedStudents` function

**Comment by psmaker** [comment]
Just checking if it this pull request is OK. It would be highly useful for us with about 120 students, it takes a bit of time to find students' name.

File: static/js/admin.js [hunk 1]
```diff
@@ -523,12 +523,13 @@ function pickedStudents(column) {
     var studentslist = students;
     var keys = [];
     var i;
-    for (i in studentslist) {
+    for (i in list) {
```
Comment (bnmnetp) [inline static/js/admin.js]: Where does list come from??

**Comment by bnmnetp** [comment]
@psmaker I don't think this is correct.  See my comment above??

**Comment by bnmnetp** [comment]
One use case for the sections list that has emerged is when a student registers twice!  It would be nice to have the username displayed along with the First and Last name to disambiguate questions like that.

**Pull request merged**